.github/workflows/codeql-analysis.yml

```diff
@@ -71,8 +71,12 @@ jobs:
     # Autobuild attempts to build any compiled languages  (C/C++, C#, or Java).
     # If this step fails, then you should remove it and run the build manually (see below)
     - name: Autobuild
+      if: ${{ !(matrix.java == '11' && matrix.language == 'java')}}
       uses: github/codeql-action/autobuild@v1
 
+    - name: Build manually
+      if: ${{ matrix.java == '11' && matrix.language == 'java' }}
+      run:  mvn clean package -f "pom.xml" -B -V -Pjdk11 -e -Dfindbugs.skip -Dcheckstyle.skip -Dpmd.skip=true -Denforcer.skip -Dmaven.javadoc.skip -DskipTests -Dmaven.test.skip.exec -Dlicense.skip=true -Drat.skip=true 
```

Comment:
Why do you skip tests when compiling with jdk11?

Reply:
This command is the same as the one executed in autobuild step. Build manually step has extra parameter to activate the profile. CodeQL analyze does not run the tests